src/utils/s3.js

```diff
@@ -0,0 +1,24 @@
+const AWS = require('aws-sdk');
+const multer = require('multer');
+const multerS3 = require('multer-s3');
+
+const s3 = new AWS.S3({
+  region: process.env.AWS_REGION,
+  accessKeyId: process.env.AWS_ACCESSKEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+});
+
+const upload = multer({
+  storage: multerS3({
+    s3: s3,
+    bucket: process.env.AWS_BUCKET_NAME,
+    contentType: multerS3.AUTO_CONTENT_TYPE,
+    key: function (req, file, cb) {
+      const uniqueId = Date.now().toString();
+      const fileName = `${uniqueId}_${file.originalname}`;
+      cb(null, fileName);
```

Comment:
unique한 key 값으로 uuid를 사용하면 좋을 것 같습니다.

```suggestion
      const uniqueId = uuid;
      cb(null, uniqueId);
```